packages/compass-welcome/src/components/web-welcome-tab.tsx

```diff
@@ -26,8 +33,229 @@ const contentBodyStyles = css({
   alignItems: 'flex-start',
 });
 
+const containerStyles = css({
+  display: 'flex',
+  alignItems: 'flex-start',
+  justifyContent: 'center',
+  margin: '0 auto',
+  minHeight: '100vh',
+  paddingTop: spacing[800], // Move content up with top padding
+});
+
+const connectingLayoutStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'flex-start', // Changed from 'center' to 'flex-start' for left alignment
+  gap: spacing[300], // Reduce gap to compress layout
+  textAlign: 'left', // Changed from 'center' to 'left'
+  maxWidth: '600px',
+  width: '100%',
+});
+
+const connectingImageContainerStyles = css({
+  display: 'flex',
+  justifyContent: 'flex-start', // Align with the text
+  marginBottom: spacing[200], // Add margin below image
+  width: 'fit-content', // Only take up as much width as needed
+  alignSelf: 'flex-start', // Align to the start of the container
+});
+
+const connectingImageStyles = css({
+  width: '334px', // Match the exact text width (333.22px rounded up)
+  height: 'auto', // Maintain aspect ratio
+  maxWidth: '100%', // Ensure it doesn't overflow container
+});
+
+const connectingTitleStyles = css({
+  width: 'fit-content', // Let the title determine its natural width
+  margin: 0, // Remove any default margins
+});
+
+const connectionListStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[300], // Increase gap for better separation of connections
+  alignItems: 'flex-start', // Changed from 'center' to 'flex-start' for left alignment
+  width: '100%',
+  marginTop: spacing[400], // Add more space above connection list
+});
+
+const connectingConnectionItemStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+  gap: spacing[200],
+  fontSize: '14px',
+  color: palette.gray.dark1, // Grayish color for connecting state
```

Comment:
We usually dont override LG default colors and fonts (especially font override seems useless here). If you have to do this (usually you really really shouldn't and design also really really shouldn't), you should take dark mode into account (the image is also not very dark mode friendly right now):

<img width="461" height="414" alt="image" src="https://github.com/user-attachments/assets/3a026af1-47e5-4643-8ec5-b4d43cae301f" />